.gitignore

```diff
@@ -2,4 +2,15 @@ node_modules
 .DS_Store
 .vscode
 .env
-temp.json
\ No newline at end of file
+temp.json
+
+
+# Ignore context files except start
+context/collections.json
+context/data.json
+context/files.json
+context/relations.json
+context/roles.json
+context/schema.json
+context/users.json
+context/completed.json
```

Comment:
We could put these stateful ones under a folder and ignore the whole folder, like `context/state`

Reply:
Done